bookingticketui/src/api/AuthenticateAPI.jsx

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios'
+
+export const BASE_URL = 'https://localhost:7089/';
+
+export const ENDPOINTS = {
+    signup: 'sign-up',
+}
+
+export const createAuthenticateAPIEndpoint = endpoint => {
+
+    let url = BASE_URL + 'api/' + 'authenticate/' + endpoint + '/';
```

Comment:
why do we need to conbine many string?

Reply:
I intend to use it as a general URL for sign-in/sign-up. Of course, I can make it into "api/authenticate/", but I think separating "api" and "authenticate" would make things clearer that this URL is only for authenticate purposes

Reply:
let url = `${BASE_URL}api/authenticate/${endpoint}/`;

Reply:
no need because of the method name is createAuthenticateAPIEndpoint so this only for authenticate